tailwind.config.ts

```diff
@@ -13,6 +13,14 @@ const config: Config = {
         'gradient-conic':
           'conic-gradient(from 180deg at 50% 50%, var(--tw-gradient-stops))',
       },
+      backgroundColor: {
+        'light': "linear-gradient(180deg, #FCD7FF 0%, #FFF 100%)",
+      }
+    },
+  },
+  variants: {
+    extend: {
+      backgroundColor: ['responsive'] // This ensures your custom class is responsive
```

Comment:
My understanding is that if you are using a css pseudo class, or similar, `hover:, md:, etc` with a custom class that you created `bg-light` you have to update the config. I believe this is why the medium size hover was not working. 

I kind of remembered this but also used ChatGPT. [Here's](https://chat.openai.com/share/dafedd9c-b680-4a3c-b22f-927ecc7dc07b) my conversation if you are curious. Notice that I started off thinking something else was wrong (the css vertical vs horizontal gradient) so it wasn't very useful at first, but when I got to the Tailwind config part, it really helped out